<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CancelPzzxSmile</class>
 <widget class="QWidget" name="CancelPzzxSmile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>取消配置中心标志</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>195</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="cancelSmileYes">
       <property name="font">
        <font>
         <family>Adobe Devanagari</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>停止笑脸</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelSmileNo">
       <property name="font">
        <font>
         <family>Adobe Devanagari</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>离开</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>50</y>
      <width>301</width>
      <height>57</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>请选择病区</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>请输入住院号</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="cancelSmileWardCombo">
         <item>
          <property name="text">
           <string>五楼全科</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>六楼风湿老年</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>七楼神三</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>八楼呼三</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cancelSmileNumInput"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
